html/about/release_notes/xdmod.html

```diff
@@ -20,10 +19,15 @@ <h2> Version 8.0.0 (2018-09-30) </h2>
                 </li>
                 <li>All XDMoD user profiles are now associated with an organization. Previously, this
                     was only required for Campus Champions.</li>
+                <li> Added support for automatically detecting / assigning a new SSO User's
```

Comment:
Why does this item have a space after the opening `<li>` and the others dont?
```suggestion
                <li>Added support for automatically detecting / assigning a new SSO User's
```